ios/Podfile.lock

```diff
@@ -2500,26 +2500,26 @@ SPEC CHECKSUMS:
   React-jsitracing: e8a2dafb9878dbcad02b6b2b88e66267fb427b74
   React-logger: 0a57b68dd2aec7ff738195f081f0520724b35dab
   React-Mapbuffer: 63913773ed7f96b814a2521e13e6d010282096ad
-  react-native-airship: 38e2596999242b68c933959d6145512e77937ac0
-  react-native-blob-util: 1ddace5234c62e3e6e4e154d305ad07ef686599b
-  react-native-cameraroll: f373bebbe9f6b7c3fd2a6f97c5171cda574cf957
+  react-native-airship: 6ab7a7974d53f92b0c46548fc198f797fdbf371f
+  react-native-blob-util: a3ee23cfdde79c769c138d505670055de233b07a
+  react-native-cameraroll: 95ce0d1a7d2d1fe55bf627ab806b64de6c3e69e9
   react-native-config: 5ce986133b07fc258828b20b9506de0e683efc1c
   react-native-document-picker: 8532b8af7c2c930f9e202aac484ac785b0f4f809
-  react-native-geolocation: f9e92eb774cb30ac1e099f34b3a94f03b4db7eb3
+  react-native-geolocation: c1c21a8cda4abae6724a322458f64ac6889b8c2b
   react-native-image-picker: f8a13ff106bcc7eb00c71ce11fdc36aac2a44440
-  react-native-key-command: 28ccfa09520e7d7e30739480dea4df003493bfe8
+  react-native-key-command: 74d18ad516037536c2f671ef0914bcce7739b2f5
   react-native-launch-arguments: 5f41e0abf88a15e3c5309b8875d6fd5ac43df49d
-  react-native-netinfo: 02d31de0e08ab043d48f2a1a8baade109d7b6ca5
-  react-native-pager-view: ccd4bbf9fc7effaf8f91f8dae43389844d9ef9fa
-  react-native-pdf: 762369633665ec02ac227aefe2f4558b92475c23
-  react-native-performance: fb21ff0c9bd7a10789c69d948f25b0067d29f7a9
-  react-native-plaid-link-sdk: 2a91ef7e257ae16d180a1ca14ba3041ae0836fbf
-  react-native-quick-sqlite: e3ab3e0a29d8c705f47a60aaa6ceaa42eb6a9ec1
-  react-native-release-profiler: 14ccdc0eeb03bedf625cf68d53d80275a81b19dd
+  react-native-netinfo: 6479e7e2198f936e5abc14a3ec4d469ccbaf81e2
+  react-native-pager-view: 9ac6bc0fb3fa31c6d403b253ee361e62ff7ccf7f
+  react-native-pdf: cd256a00b9d65cb1008dcca2792d7bfb8874838d
+  react-native-performance: 1aa5960d005159f4ab20be15b44714b53b44e075
+  react-native-plaid-link-sdk: 93870f8cd1de8e0acca5cb5020188bdc94e15db6
+  react-native-quick-sqlite: f7b9f578b8b3b608dc742240b0103faae5b61f63
+  react-native-release-profiler: 42fc8e09b4f6f9b7d14cc5a15c72165e871c0918
   react-native-render-html: 96c979fe7452a0a41559685d2f83b12b93edac8c
```

Comment:
@robertKozik I think these changes are not needed here.
We can ignore all the changes and update only RNLiveMarkdown (0.1.70) and RNLiveMarkdown/common (0.1.70).

Reply:
Yeah, I'm not sure how they came up here. Anyways new version is already pushed without these changes